### PR TITLE
add :getFrame()

### PR DESCRIPTION
**Pull request by flamendless**

### CODE REVIEW

File: peachy.lua [hunk 1]
```diff
@@ -127,6 +127,15 @@ function peachy:setFrame(frame)
   self.frameTimer = cron.after(self.frame.duration, self.nextFrame, self)
 end
 
+--- Get the current frame of the current animation
+-- @usage
+-- Get the 2nd frame
```
Comment (josh-perry) [inline peachy.lua]: 2nd?

File: peachy.lua [hunk 2]
```diff
@@ -127,6 +127,15 @@ function peachy:setFrame(frame)
...
+-- local f = sound:getFrame()
+--
+function peachy:getFrame()
+	return self.frameIndex
```
Comment (josh-perry) [inline peachy.lua]: Can you make this tab into 2 spaces to match the rest of the file?

**Comment by josh-perry** [comment]
Can you also rebase to remove 40b363e?

**Comment by flamendless** [comment]
There, i just resolved conflict.

**Pull request closed without merging**